ui/app/src/components/shared/ResourceCard.tsx

```diff
@@ -110,7 +115,7 @@ export const ResourceCard: React.FunctionComponent<ResourceCardProps> = (props:
           <Stack
             styles={cardStyles}
             aria-labelledby={`card-${props.resource.id}`}
-            onClick={() => {if (!authNotProvisioned) goToResource()}}
+            onClick={() => {if (enableClickOnCard) goToResource()}}
```

Comment:
I can't remember but does the card grow on hover / depress on click - i.e. indicate that it's clickable even when the onClick condition evaluates to false? I suspect so and it might be worth disabling those cardStyles if this condition is false

Reply:
Done.